tweepy/user.py

```diff
@@ -103,6 +105,7 @@ def __init__(self, data):
         self.public_metrics = data.get("public_metrics")
         self.url = data.get("url")
         self.verified = data.get("verified")
+        self.verified = data.get("verified_type")
```

Comment:
This would override the existing `verified` attribute/field and doesn't match the documented attribute, field, or slots name.
```suggestion
        self.verified_type = data.get("verified_type")
```